pootle/apps/virtualfolder/utils.py

```diff
@@ -249,3 +249,27 @@ def get_stats(self, user=None):
         if self.show_all_to(user):
             return self.all_vf_stats
         return self.vf_stats
+
+    @persistent_property
+    def all_checks_data(self):
+        checks = {}
+        cd = (
+            self.filter_data(self.checks_data_model)
+                .values_list("store__vfolders", "name")
+                .annotate(Sum("count")))
+        for vf, name, count in cd:
+            checks[vf] = checks.get(vf, {})
+            checks[vf][name] = count
+        return checks
+
+    @persistent_property
+    def checks_data(self):
+        checks = {}
+        cd = self.filter_accessible(
+            self.filter_data(self.checks_data_model))
+        cd = (cd.values_list("store__vfolders", "name")
+                .annotate(Sum("count")))
+        for vf, name, count in cd:
+            checks[vf] = checks.get(vf, {})
+            checks[vf][name] = count
+        return checks
```

Comment:
yep - the parent class does that - but here the filtering needs to happen at a diff point - i can factor out still tho